.github/workflows/api_tests.yaml

```diff
@@ -7,7 +7,7 @@ jobs:
     apicurltests:
         strategy:
             matrix:
-                php-version: [ '7.2', '8.0' ]
+                php-version: [ '8.1', '8.2' ]
```

Comment:
Let's try and add 8.3 & 8.4 directly (holds for all changes in this PR)

Reply:
Let's go !